ale/drivers/mdis_driver.py

```diff
@@ -109,6 +109,13 @@ def instrument_id(self):
         """
         return self.id_lookup[self.label['INSTRUMENT_ID']]
 
+    @property
+    def line_exposure_duration(self):
+        try:
+            return self.label['EXPOSURE_DURATION'].value * 0.001
+        except:
+            return np.nan
```

Comment:
remove the except. Apparently the C++ JSON loader doesn't like NaN as a value and fails parsing.  If `ExposureDuration` is not there, let it fail.